data/schema/achievements.cfg

```diff
@@ -0,0 +1,23 @@
+[wml_schema]
+	{./macros.cfg}
+	{./types/basic.cfg}
+	[tag]
+		name="root"
+		[tag]
+			name="achievement_group"
```

Comment:
min and max default to 0 and 1, respectively. At least the latter is clearly wrong.

Reply:
Fixed as well.